pandas/tests/io/excel/conftest.py

```diff
@@ -42,26 +39,3 @@ def read_ext(request):
     Valid extensions for reading Excel files.
     """
     return request.param
-
-
-# Checking for file leaks can hang on Windows CI
-@pytest.fixture(autouse=not is_platform_windows())
-def check_for_file_leaks():
-    """
-    Fixture to run around every test to ensure that we are not leaking files.
-
-    See also
-    --------
-    _test_decorators.check_file_leaks
-    """
-    # GH#30162
-    psutil = td.safe_import("psutil")
-    if not psutil:
-        yield
-
-    else:
-        proc = psutil.Process()
-        flist = proc.open_files()
-        yield
-        flist2 = proc.open_files()
```

Comment:
documenting in case we wind up keeping this: looks like all of the flaky failures have raddr.port of either 5432 or 3306 (mostly 5432), which correspond to postgres and mysql defaults, respectively.  We could filter those out here

Reply:
FWIW I have seen the port 443 show up a few times too